.github/workflows/ci.yml

```diff
@@ -816,6 +842,66 @@ jobs:
           fail_on_unmatched_files: true
           prerelease: ${{ contains(steps.semver.outputs.group1, '-') }}
 
+  upload-sentry:
+    name: sentry upload
```

Comment:
@SleepySquash seems like better fit for `Deploy` section.

Could be named `deploy (Sentry, symbols)`.